### ISSUE TITLE
Fails to run tests when path contains a space

### ISSUE THREAD

**Issue by MikeHopley** [Codeception/Codeception#2762]
If the test file's path contains a space, then the test will not run, as Codeception cannot find the file.

For example, say we execute `codecept run tests/acceptance/someCept`, where the full path is `C:/folder1/some path/app/tests/acceptance/someCept`. Codeception will fail with an error:

```
[Codeception\Exception\TestParseException]
Couldn't parse test 'C:/folder1/some path/app/tests/acceptance/someCept.php'
Could not open input file: C:/folder1/some
```

This is on Windows 10, and Codeception 2.1.5. The error is **not** present in Codeception 2.1.4.

**Comment by DavertMik** [comment]
Fixed in 2.1 branch. Please wait for 2.1.6 or download 2.1

**Comment by MikeHopley** [comment]
Thanks Davert!
